src/Microsoft.Health.Fhir.TemplateManagement/Client/OrasClient.cs

```diff
@@ -17,46 +17,65 @@ namespace Microsoft.Health.Fhir.TemplateManagement.Client
     public class OrasClient : IOrasClient
     {
         private readonly string _imageReference;
+        private readonly Regex _digestRx = new Regex("(?<rule>[A-Za-z][A-Za-z0-9]*([+.-_][A-Za-z][A-Za-z0-9]*)*):(?<digest>[0-9a-fA-F]{32,})");
 
         public OrasClient(string imageReference)
         {
             EnsureArg.IsNotNull(imageReference, nameof(imageReference));
 
             _imageReference = imageReference;
+            if (string.IsNullOrEmpty(Environment.GetEnvironmentVariable("ORAS_CACHE")))
+            {
+                Environment.SetEnvironmentVariable("ORAS_CACHE", ".oras/cache");
+            }
         }
 
         public async Task PullImageAsync(string outputFolder)
         {
             string command = $"pull  \"{_imageReference}\" -o \"{outputFolder}\"";
-            await OrasExecutionAsync(command, Directory.GetCurrentDirectory());
+            string output = await OrasExecutionAsync(command, Directory.GetCurrentDirectory());
+            Console.WriteLine(output);
```

Comment:
Shall we remove this output?

Reply:
better to output the message to customer?

Reply:
Since this is a library, I feel it's better to return an object like OciResult or OciOperationResult. It's for customers' choice to output or log the result. In your case, you can write output in the Tool exe.

Reply:
Thanks, I add an OrasOperationResult to store the output message.